proposal/README.md

```diff
@@ -1,12 +1,12 @@
 # Introduction
 
-This document proposes, **flytectl** as one singular CLI that interacts with flyteadmin service. It is proposed to write the CLI in **Golang** and would support both gRPC and REST endpoints of
+This document proposes, **Flytectl** as one singular CLI that interacts with FlyteAdmin service. It is proposed to write the CLI in **Golang** and would support both gRPC and REST endpoints of
 FlyteAdmin. We will start with gRPC endpoint as the client is easily generated and in future we should investigate generation of a Swagger based REST client from the gRPC specification. As we build
 more SDK's in different languages we should support a common way of interacting with the API. This does no mean that some SDK's may provide native ways of interacting with the Admin API (for e.g.
-flytekit), but the intention is that we will eventually replace **flytekit/flyte-cli** with flytectl exclusively.
+Flytekit), but the intention is that we will eventually replace **Flytekit/Flytecli** with Flytectl exclusively.
 
-We also recommend that the design of FlyteCTL is careful and it could helps us with delivering user features faster without having to rely on the UI. FlyteCTL with follow standard oauth2 for
-authentication already supported by flyteAdmin. Moreover, FlyteCTL should be readily available on almost any platform - OSX, Linux and Windows. We will strive to keep it relatively lean and fast.
+We also recommend that the design of Flytectl is careful and it could helps us with delivering user features faster without having to rely on the UI. Flytectl with follow standard oauth2 for
```

Comment:
```suggestion
We also recommend that the design of Flytectl is careful, and it could help us deliver user features faster without having to rely on the UI. Flytectl will follow standard oauth2 for
```